### DISCUSSION TITLE
دسترسی

### DISCUSSION THREAD

**Discussion by Saeedazimi69** [community/community#137551] [Mobile]
### Select Topic Area

General

### GitHub Feature Area

Mobile

### Body

پشتیبانی وحمایت ازگوشی سامسونگ گلکسی جی ۶۰۰

**Comment by elisealix22** [comment]
Hi @Saeedazimi69 , it seems like the Samsung Galaxy J6 is a supported device in our list! Do you know the Android version you are running on your device?
